### PR TITLE
Ghpython moar artists

### PR DESCRIPTION
**Pull request by gonzalocasas**
Added a bunch of missing Grasshopper python artists:
```markdown
Object      Artist              Rhino    Blender  GH       Plotters
------------------------------------------------------------------
Box         BoxArtist           [x]      [x]      [x]      [n/a]
Capsule     CapsuleArtist       [x]      [x]      [x]      [n/a]
Circle      CircleArtist        [x]      [ ]      [x]      [x]
Cone        ConeArtist          [x]      [x]      [x]      [n/a]
Cylinder    CylinderArtist      [x]      [x]      [x]      [n/a]
Ellipse     EllipseArtist       [ ]      [ ]      [ ]      [x]
Frame       FrameArtist         [x]      [x]      [x]      [n/a]
Line        LineArtist          [x]      [ ]      [x]      [x]
Mesh        MeshArtist          [x]      [x]      [x]      [x]
Network     NetworkArtist       [x]      [x]      [x]      [x]
Plane       PlaneArtist         [x]      [ ]      [ ]      [n/a]
Point       PointArtist         [x]      [ ]      [x]      [x]
Polygon     PolygonArtist       [x]      [ ]      [x]      [x]
Polyhedron  PolyhedronArtist    [x]      [x]      [x]      [n/a]
Polyline    PolylineArtist      [x]      [ ]      [x]      [x]
RobotModel  RobotModelArtist    [x]      [x]      [x]      [n/a]
Segment     SegmentArtist       [ ]      [ ]      [ ]      [x]
Sphere      SphereArtist        [x]      [x]      [x]      [n/a]
Torus       TorusArtist         [x]      [x]      [x]      [n/a]
Vector      VectorArtist        [x]      [ ]      [x]      [x]
VolMesh     VolMeshArtist       [x]      [x]      [x]      [n/a]
```

The artist test looks like this now:
image

### What type of change is this?

- [x] Bug fix in a **backwards-compatible** manner.
- [x] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [ ] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [x] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [x] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [x] I added new functions/classes and made them available on a second-level import, e.g. `compas.datastructures.Mesh`.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by tomvanmele — APPROVED** [review]
LGTM

perhaps we have to revisit some of the available drawing options for primitives (like the `show` options for polygon), but that should probably go in a separate PR...

File: src/compas_ghpython/artists/polygonartist.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            Default is ``False``.
+        show_edges : bool, optional
+            Default is ``False``.
+        show_face : bool, optional
+            Default is ``True``.
+
+        Returns
+        -------
+        list
+            The Rhino points, lines and faces.
```
Comment (beverlylytle) [inline src/compas_ghpython/artists/polygonartist.py]:
```suggestion
            The Rhino points, lines and face.
```

File: src/compas_ghpython/artists/boxartist.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
```
Comment (beverlylytle) [inline src/compas_ghpython/artists/boxartist.py:3]: This is very unlike you, Gonzalo.  Is everything ok? ;)

File: src/compas_ghpython/artists/vectorartist.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        Parameters
+        ----------
+        point : [float, float, float] or :class:`compas.geometry.Point`, optional
+            Point of application of the vector.
+            Default is ``Point(0, 0, 0)``.
+
+        Returns
+        -------
+        list
+            The Rhino lines.
```
Comment (beverlylytle) [inline src/compas_ghpython/artists/vectorartist.py]:
```suggestion
            The Rhino line and endpoints, if requested.
```

File: src/compas_ghpython/utilities/drawing.py [hunk 4]
```diff
@@ -26,6 +29,11 @@
 from Rhino.Geometry import Vector3f
 from Rhino.Geometry import Point2f
 
+try:
+    from Rhino.Geometry import MeshNgon
+except ImportError:
+    MeshNgon = False
```
Comment (beverlylytle) [inline src/compas_ghpython/utilities/drawing.py:35]: Why does this need to be wrapped in a `try`-`except` clause?

**Review by beverlylytle — COMMENTED** [review]
Looks good!

File: src/compas_ghpython/artists/boxartist.py (hunk 2 above)
Comment (gonzalocasas) [reply src/compas_ghpython/artists/boxartist.py:3]: 😆

File: src/compas_ghpython/utilities/drawing.py (hunk 4 above)
Comment (tomvanmele) [reply src/compas_ghpython/utilities/drawing.py:35]: not supported on all versions of Rhino

File: src/compas_ghpython/artists/__init__.py [hunk 5]
```diff
@@ -155,13 +191,22 @@ def new_artist_gh(cls, *args, **kwargs):
     'GHArtist',
     'PrimitiveArtist',
```
Comment (beverlylytle) [inline src/compas_ghpython/artists/__init__.py]: This one doesn't appear to have been imported.

Comment (gonzalocasas) [reply src/compas_ghpython/artists/__init__.py]: fixed! https://github.com/compas-dev/compas/pull/914/commits/ebf2071428230b9ec8088f5b5a95d97004f42152

**Pull request merged**